root/static/scripts/release-editor/actions.js

```diff
@@ -270,6 +270,43 @@ const actions = {
     track.nameModified(false);
   },
 
+  onTrackInputKeyDown(track, event) {
+    /*
+     * If a bare up or down arrow key event occurs while the cursor is
+     * at the beginning or end of the input, move the focus up or down.
+     */
+    if (
+      (event.key === 'ArrowUp' || event.key === 'ArrowDown') &&
+      !event.altKey && !event.ctrlKey && !event.metaKey &&
+      !event.shiftKey
+    ) {
+      const up = event.key === 'ArrowUp';
+      const input = event.target;
+      if (input.selectionStart === input.selectionEnd && (
+        (up && input.selectionStart === 0) ||
+        (!up && input.selectionStart === input.value.length)
+      )) {
+        const row = input.closest('tr');
+        const rows = [...row.parentNode.children];
+        const rowIndex = rows.indexOf(row);
+        const newRowIndex = rowIndex + (up ? -1 : 1);
+        if (newRowIndex >= 0 && newRowIndex < rows.length) {
```

Comment:
I considered making this also jump between mediums, but I'm not sure if it's worth the added complexity. I also wasn't sure if it'd feel to gross to have an e.g. `document.querySelector('table.medium')` call in this file.